### PR TITLE
introduce routines concept

### PR DESCRIPTION
**Pull request by DidierRLopes**
@deeleeramone

### CODE REVIEW

File: routines/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ROUTINES
+
+This folder introduces the concept of routines, which are meant for the user to develop and run at once. The notebook reports is a better approach to this idea, but currently the development of this is not trivial. On the other hand, the development of this is trivial as long as the user is familiar with our terminal interface.
+
+Try out with:
+```
+python terminal.pyp routines/didier_gme_job.gst
```
Comment (jmaslek) [inline routines/README.md:7]: typo

File: routines/didier_gme_job.gst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+rating
+pt
+sec
+../ta
+bbands
+rsi
+fib
+../pred
+mc
+lstm -l 3 --epochs 200 -d 20
```
Comment (jmaslek) [inline routines/didier_gme_job.gst:22]: if people are going to copy this -- change epochs to 20 lol

**Comment by jmaslek** [comment]
Only other comment is that I noticed MPL yells if you have 20 windows open lol.  Not sure if you wanna note that or not.

**Review by jmaslek — APPROVED** [review]

**Comment by DidierRLopes** [comment]
> Only other comment is that I noticed MPL yells if you have 20 windows open lol. Not sure if you wanna note that or not.

It yells but it works right?

**Comment by jmaslek** [comment]
> > Only other comment is that I noticed MPL yells if you have 20 windows open lol. Not sure if you wanna note that or not.
> 
> It yells but it works right?


yup

**Review by jmaslek — APPROVED** [review]

**Comment by DidierRLopes** [comment]
> > > Only other comment is that I noticed MPL yells if you have 20 windows open lol. Not sure if you wanna note that or not.
> > 
> > 
> > It yells but it works right?
> 
> yup

We can live with that I think 😛

**Comment by deeleeramone** [comment]
python terminal.pyp routines/didier_gme_job.gst

Candle -m did not launch a window 

RSI and BBands did not launch a window

**Pull request merged**